lhotse/bin/lhotse

```diff
@@ -4,9 +4,11 @@ Use this script like:
 
 $ lhotse --help
 $ lhotse make-feats --help
-$ lhotse make-feats --compressed audio_manifest.yml mfcc_dir
-
+$ lhotse make-feats --compressed audio_manifest.yml mfcc_dir/
 $ lhotse default-feature-config feat-conf.yml
```

Comment:
maybe make default-feature-config `write-default-feature-config` ?  Otherwise it looks as if you might be attempting to change a default somehow, like git config.